modules/collectd_vsphere/collectd_vsphere.tf

```diff
@@ -62,16 +62,18 @@ data "template_file" "collectd_vsphere_upstart" {
 
 resource "null_resource" "collectd_vsphere" {
   triggers {
-    version                               = "${var.version}"
-    config_signature                      = "${sha256(file(var.config_path))}"
-    install_script_signature              = "${sha256(data.template_file.collectd_vsphere_install.rendered)}"
-    librato_creds_signature               = "${sha256(data.template_file.librato_conf.rendered)}"
-    collectd_config_signature             = "${sha256(data.template_file.collectd_conf.rendered)}"
-    collectd_vsphere_init_signature       = "${sha256(data.template_file.collectd_vsphere_upstart.rendered)}"
-    collectd_network_token_file_signature = "${sha256(data.template_file.collectd_network_token_file.rendered)}"
-    collectd_snmp_config_signature        = "${sha256(data.template_file.snmp_conf.rendered)}"
-    name                                  = "${var.env}-${var.index}"
-    host_id                               = "${var.host_id}"
+    version                                              = "${var.version}"
+    config_signature                                     = "${sha256(file(var.config_path))}"
+    install_script_signature                             = "${sha256(data.template_file.collectd_vsphere_install.rendered)}"
+    librato_creds_signature                              = "${sha256(data.template_file.librato_conf.rendered)}"
+    collectd_config_signature                            = "${sha256(data.template_file.collectd_conf.rendered)}"
+    collectd_vsphere_collectd_network_username_signature = "${sha256(var.collectd_vsphere_collectd_network_user)}"
```

Comment:
The collectd network username is no longer a secret, so does that mean that it should be used directly instead of its' `sha256` value?

Reply:
good catch!